lib/ansible/modules/cloud/docker/docker_stack.py

```diff
@@ -18,7 +18,7 @@
 author: "Dario Zanzico (@dariko)"
 short_description: docker stack module
 description:
--   Manage docker stacks using the 'docker stack' command
+-   Manage docker stacks using the 'docker_stack' command
```

Comment:
This is an incorrect change. This refers to the command line that is used which is `docker stack`

Reply:
Oh, you're right. Apologies.